xarray/tests/test_coding_times.py

```diff
@@ -161,7 +163,7 @@ def test_decode_cf_datetime_non_iso_strings():
 @pytest.mark.skipif(not has_cftime_or_netCDF4, reason='cftime not installed')
 @pytest.mark.parametrize(
     ['calendar', 'enable_cftimeindex'],
-    product(coding.times._STANDARD_CALENDARS, [False, True]))
+    product(_STANDARD_CALENDARS, [False, True]))
```

Comment:
For any future pytesters - no need to make the `product` - put a `@...parametrize` for each param and pytest will do the product for you